<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable" tests="18" time="140.511" ignored="0" failures="3" timestamp="26 May 2021 12:25:37 GMT" skipped="0" hostname="IN-L2504" errors="2">
  <testcase name="testGetNotificationsForCardStatementAvailable" time="8.604" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable"/>
  <testcase name="testGetNotificationsForCardStatementAvailable" time="0.099" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.yodlee.insights.yodleeApi.utils.BoardHelper.createViewRules(BoardHelper.java:692)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable.testGetNotificationsForCardStatementAvailable(TestGetInsightsForCardStatementAvailable.java:209)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- testGetNotificationsForCardStatementAvailable -->
  <testcase name="testGetNotificationsForCardStatementAvailable" time="9.062" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable"/>
  <testcase name="testGetNotificationsForCardStatementAvailable" time="8.517" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable">
    <failure type="junit.framework.AssertionFailedError" message="Expected Number of Insights are not generated for All Accounts, Expected = 2 Insights Actual = 1 Insights">
      <![CDATA[junit.framework.AssertionFailedError: Expected Number of Insights are not generated for All Accounts, Expected = 2 Insights Actual = 1 Insights
	at junit.framework.Assert.fail(Assert.java:57)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable.testGetNotificationsForCardStatementAvailable(TestGetInsightsForCardStatementAvailable.java:315)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCardStatementAvailable -->
  <testcase name="testGetNotificationsForCardStatementAvailable" time="0.119" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.yodlee.insights.yodleeApi.utils.BoardHelper.createViewRules(BoardHelper.java:692)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable.testGetNotificationsForCardStatementAvailable(TestGetInsightsForCardStatementAvailable.java:209)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- testGetNotificationsForCardStatementAvailable -->
  <testcase name="testGetNotificationsForCardStatementAvailable" time="8.522" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable"/>
  <testcase name="testGetNotificationsForCardStatementAvailable" time="8.651" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable"/>
  <testcase name="testGetNotificationsForCardStatementAvailable" time="8.625" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable"/>
  <testcase name="testGetNotificationsForCardStatementAvailable" time="8.551" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable"/>
  <testcase name="testGetNotificationsForCardStatementAvailable" time="9.226" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable"/>
  <testcase name="testGetNotificationsForCardStatementAvailable" time="8.686" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable"/>
  <testcase name="testGetNotificationsForCardStatementAvailable" time="9.603" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable"/>
  <testcase name="testGetNotificationsForCardStatementAvailable" time="8.729" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable"/>
  <testcase name="testGetNotificationsForCardStatementAvailable" time="8.843" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable"/>
  <testcase name="testGetNotificationsForCardStatementAvailable" time="8.557" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable">
    <failure type="junit.framework.AssertionFailedError" message="Expected Number of Insights are not generated for All Accounts, Expected = 2 Insights Actual = 1 Insights">
      <![CDATA[junit.framework.AssertionFailedError: Expected Number of Insights are not generated for All Accounts, Expected = 2 Insights Actual = 1 Insights
	at junit.framework.Assert.fail(Assert.java:57)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable.testGetNotificationsForCardStatementAvailable(TestGetInsightsForCardStatementAvailable.java:315)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCardStatementAvailable -->
  <testcase name="testGetNotificationsForCardStatementAvailable" time="8.617" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable"/>
  <testcase name="testGetNotificationsForCardStatementAvailable" time="8.698" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable"/>
  <testcase name="testGetNotificationsForCardStatementAvailable" time="8.802" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable">
    <failure type="junit.framework.AssertionFailedError" message="Expected  1 insights triggered But found Empty {}">
      <![CDATA[junit.framework.AssertionFailedError: Expected  1 insights triggered But found Empty {}
	at junit.framework.Assert.fail(Assert.java:57)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable.testGetNotificationsForCardStatementAvailable(TestGetInsightsForCardStatementAvailable.java:315)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCardStatementAvailable -->
</testsuite> <!-- com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCardStatementAvailable -->
